src/api.py

```diff
@@ -25,7 +27,32 @@ def home():
         return 'WELCOME TO CABINET'
 
 
-    @app.route('/blob', methods=['GET', 'POST', 'PUT'])
+    @app.route('/hosts', methods=['GET'])
+    def hosts():
+        with open('config/config.yaml','r') as file:
+            config_dict = yaml.safe_load(file) 
+        hosts = list(config_dict['save_hosts'].keys())
+        return Response(body={'hosts':hosts}).json() 
+        
+
+    @app.route('/paths', methods=['GET'])
```

Comment:
rename to "storage location" or "location url"

Reply:
done: storage_locations